plugins/woocommerce/includes/class-wc-cli.php

```diff
@@ -29,6 +29,8 @@ private function includes() {
 		require_once dirname( __FILE__ ) . '/cli/class-wc-cli-tool-command.php';
 		require_once dirname( __FILE__ ) . '/cli/class-wc-cli-update-command.php';
 		require_once dirname( __FILE__ ) . '/cli/class-wc-cli-tracker-command.php';
+		$cli_runner = wc_get_container()->get( \Automattic\WooCommerce\DataBase\Migrations\CustomOrderTable\CLIRunner::class );
+		WP_CLI::add_command( 'wc cot', $cli_runner );
```

Comment:
This goes slightly against the pattern we have here, where normally the COT cli runner would define a `register_commands` method and run the `WP_CLI::add_command` over there, then hook it down in the `hooks` method to `after_wp_load` instead of running on instantiation. Do you think it would be worth changing it or is it just unnecessary abstraction overhead?

Reply:
Thanks for pointing out, changed how we register these method in https://github.com/woocommerce/woocommerce/pull/32938/commits/124fdd37f35389dd7ac8ba92255bb159dbc26b9b